src/main/java/org/skriptlang/skript/bukkit/breeding/elements/CondCanBreed.java

```diff
@@ -17,20 +22,38 @@
 		"\tsend \"Turns out %event-entity% is not breedable. Must be a Skript user!\" to player"
 })
 @Since("2.10")
-public class CondCanBreed extends PropertyCondition<LivingEntity> {
+public class CondCanBreed extends PropertyCondition<LivingEntity> implements SyntaxRuntimeErrorProducer {
 
-	static {
-		register(CondCanBreed.class, PropertyType.CAN, "(breed|be bred)", "livingentities");
+	public static void register(SyntaxRegistry registry) {
+		register(registry, CondCanBreed.class, PropertyType.CAN, "(breed|be bred)", "livingentities");
+	}
+
+	private Node node;
+
+	@Override
+	public boolean init(Expression<?>[] expressions, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		node = getParser().getNode();
+		return super.init(expressions, matchedPattern, isDelayed, parseResult);
 	}
 
 	@Override
 	public boolean check(LivingEntity entity) {
-		return entity instanceof Breedable breedable && breedable.canBreed();
+		if (entity instanceof Breedable breedable) {
+			return breedable.canBreed();
+		} else {
+			warning("An entity passed through wasn't breedable, and thus returned false by default.");
```

Comment:
apply this same logic to all the warnings in this pr. are they really warning about something productive?